pennylane/gradients/parameter_shift.py

```diff
@@ -1281,16 +1280,31 @@ def _param_shift_new(
     to use during autodifferentiation:
 
     >>> dev = qml.device("default.qubit", wires=2)
-    >>> @qml.qnode(dev, gradient_fn=qml.gradients.param_shift)
+    >>> @qml.qnode(dev, interface="autograd", diff_method="parameter-shift")
     ... def circuit(params):
     ...     qml.RX(params[0], wires=0)
     ...     qml.RY(params[1], wires=0)
     ...     qml.RX(params[2], wires=0)
-    ...     return qml.expval(qml.PauliZ(0)), qml.var(qml.PauliZ(0))
+    ...     return qml.expval(qml.PauliZ(0))
     >>> params = np.array([0.1, 0.2, 0.3], requires_grad=True)
     >>> qml.jacobian(circuit)(params)
-    tensor([[-0.38751725, -0.18884792, -0.38355708],
-            [ 0.69916868,  0.34072432,  0.69202365]], requires_grad=True)
+    tensor([-0.38751725, -0.18884792, -0.38355708], requires_grad=True)
+
+    For QNode integration with Autograd respectively TensorFlow and multiple measurements, the results need to be
+    stacked before differentiate them. It is because those frameworks do not support differentiating tuple. On the other
+    side no additional requirement is needed for Jax and Torch.
+
+    >>> import jax
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> @qml.qnode(dev, interface="jax", diff_method="parameter-shift")

```

Comment:
Does this example require enabling the new return type changes? maybe that should be in the doc string then?

Reply:
These docstrings will become the default, so we're not adding the enable return statements explicitly.

Reply:
It is written in the function docstring, that it is for the new return type. Eventually it will replace the actual one.